_posts/2020-06-07-higher-order-hydras.markdown

```diff
@@ -0,0 +1,491 @@
+There's a long descent until the \\\(\omega\\\) labels are gone though, and still much longer to termination.
+
+</div>
+
+</div>
+
+# A program implementing Amiko hydras
+{: #amiko-hydras-program}
+
+{% include collapser.markdown %}
+
+<div>
+
+To save myself the trouble of needing to do larger hydras by hand, and to allow you to play with Amiko hydras, I wrote a program which includes all the related functionality: [amiko_hydra.py](/assets/hydras/amiko_hydra.py).
+Please keep in mind I do not yet know the comparison rules for Amiko hydras, so I used the Buchholz hydras comparison rules as a placeholder.
+It's not as clean as my usual code, but please forgive me, I was very excited to share this with the world so I took some shortcuts.
+
+</div>
+
+# Higher order hydras
+{: #higher-order-hydras}
+
+{% include collapser.markdown %}
+
+<div>
+
+This whole time I've been avoiding mention of "higher order hydras".
+A higher order theory should quantify over and work with the objects of the lower order theory, for example, first order arithmetic quantifies over integers, and second order arithmetic can now quantify over sets of integers (and thus relations on integers and real numbers).
+While it may be true that we used a higher order technique to get to this hydra construction, the hydras themselves are first order trees and the evolution rules are all first order computable,
+which I understand are very different things, but the point is I can't find anything second order about the hydras themselves which would allow me to refer to them as second order.
+
+For the time being, Amiko hydras are just a hydra construction that is stronger than Buchholz hydras and introduces new techniques.
+
+</div>
+
+# Stronger hydra constructions beyond Amiko hydras
+{: #beyond-amiko-hydras}
+
+{% include collapser.markdown %}
+
+<div>
+
+Amiko hydras are by no means the end.
+I've already introduced externalize and close, so I acknowledge that you can boost further with more iterations of externalize and close.
+The reason I don't bother just presenting those instead is because Amiko hydras are in some sense the minimal example for this technique,
+and stacking on more externalize and close iterations does not necessarily make it more interesting.
+It would make it more confusing though.
+
+I believe the next stronger hydra worthy of study must introduce something new.
+Buchholz introduced labels.
+I introduced externalize and close as higher order transformations.
+What comes next may be a higher order closure over the hydra constructions reachable by externalize and close, or perhaps something more novel and creative that I have not even conceived.
+
+Though not a hydra unless it's perhaps a flattened hydra, I want to point out the [Bashicu matrix system](https://googology.wikia.org/wiki/Bashicu_matrix_system), which is one of the contenders for being a next stronger system.
```

Comment:
Change "point out" to "bring attention to".

Reply:
sure